internal/commands/secrets/delete_inputs.go

```diff
@@ -0,0 +1,83 @@
+package secrets
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	flagSecret      = "secret"
+	flagSecretShort = "s"
+	flagSecretUsage = "set the list of secrets to delete by ID or Name"
+)
+
+type deleteInputs struct {
+	cli.ProjectInputs
+	secrets []string
+}
+
+func (i *deleteInputs) Resolve(profile *cli.Profile, ui terminal.UI) error {
+	if err := i.ProjectInputs.Resolve(ui, profile.WorkingDirectory); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (i *deleteInputs) resolveSecrets(ui terminal.UI, allSecrets []realm.Secret) ([]realm.Secret, error) {
+	var toDelete []realm.Secret
+
+	if len(i.secrets) == 0 {
```

Comment:
can we flip this structure a bit for readability purposes? something like:

```
func (i *deleteInputs) resolveSecrets(ui terminal.UI, secrets []realm.Secret) ([]realm.Secret, error) {
  if len(i.secrets) > 0 {
    resolved := make([]realm.Secret, len(i.secrets))
    // ...
    return resolved, nil
  }

  // no "else" needed here since we return above, less indentation for the next code path
}
```